### PR TITLE
Fix path to write the env

### PR DESCRIPTION
**Pull request by joecohens**
PING @GrahamCampbell 

Closes https://github.com/CachetHQ/Cachet/issues/1486

### CODE REVIEW

File: app/Http/Controllers/SetupController.php [hunk 1]
```diff
@@ -222,12 +222,13 @@ public function postStep3()
     protected function writeEnv($key, $value)
     {
         $path = app()->environmentFile();
+        $basePath = base_path($path);
 
         try {
             (new Dotenv(app()->environmentPath(), $path))->load();
```
Comment (GrahamCampbell) [inline app/Http/Controllers/SetupController.php]: Why does the path work here but not else where?

Comment (joecohens) [inline app/Http/Controllers/SetupController.php]: No

Comment (joecohens) [inline app/Http/Controllers/SetupController.php]: Not really sure :( but we had an error on writing the file

Comment (GrahamCampbell) [inline app/Http/Controllers/SetupController.php]: Oh, wait, i see. Don't call base_path. One sec...

**Comment by GrahamCampbell** [comment]
Also, we can't assume `base_path` is where the `.env` file is located because laravel supports changing that location.

**Comment by GrahamCampbell** [comment]
I've got a slightly different fix on the way. :)

**Pull request closed without merging**